TypeCobol/Compiler/Diagnostics/Cobol2002Checker.cs

```diff
@@ -49,7 +49,7 @@ public static void CheckTypedef(DataTypeDescriptionEntry typedef, CodeElementsPa
 #if EUROINFO_LEGACY_TYPEDEF
             if (typedef.RestrictionLevel != RestrictionLevel.STRICT)
             {
-                string message = "Custom EI rule : Only TYPEDEF STRICT is allowed.";
+                string message = $"Custom EI rule : Only TYPEDEF STRICT is allowed in definition of type {typedef.DataName.Name}.";
```

Comment:
Reason is to avoid duplicate errors; see the previous conversation:
Ideally, we should avoid duplicate errors. I guess that what is happening here is that we get one error from the copy and one from the usage of the type. So to avoid that, the checker should only raise one error but not both.
----adding to the message "Syntax error : Custom EI rule : Only TYPDEF STRICT is allowed" "in definition of type <name_of_the_typedef>.".

Reply:
Maybe use this error message : "type <type name> must be marked as STRICT." ?